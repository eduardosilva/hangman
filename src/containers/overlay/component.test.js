import React from 'react'
import renderer from 'react-test-renderer'
import Overlay from './component'

it('should render div win', () => {
  const letter = renderer
    .create(<Overlay win lost={false} word={[]} onButtonClick={() => 'clicked'} />)
    .toJSON()
  expect(letter).toMatchSnapshot()
})

it('should render div lost', () => {
  const letter = renderer
    .create(<Overlay win={false} lost word={[]} onButtonClick={() => 'clicked'} />)
    .toJSON()
  expect(letter).toMatchSnapshot()
})
