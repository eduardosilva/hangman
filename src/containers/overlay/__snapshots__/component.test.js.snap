// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render div lost 1`] = `
<div
  className={undefined}
>
  <h1>
    You Lost
  </h1>
  <h2
    className={undefined}
  >
    Word is: 
  </h2>
  <button
    onClick={[Function]}
  >
    New word
  </button>
</div>
`;

exports[`should render div win 1`] = `
<div
  className={undefined}
>
  <h1>
    You Win
  </h1>
  <h2
    className={undefined}
  >
    Word is: 
  </h2>
  <button
    onClick={[Function]}
  >
    New word
  </button>
</div>
`;
